tests/modules/projects/test_models.py

```diff
@@ -110,8 +111,13 @@ def test_project_permission(db, regular_user):
     logging.info(temp_user.project_membership_enrollments)
     logging.info(temp_proj.user_membership_enrollments)
 
-    assert temp_user.has_permission_to_read(temp_encounter) is True
-    assert regular_user.has_permission_to_read(temp_encounter) is False
+    assert (
```

Comment:
we can simply use `assert  temp_user.can_perform_action(temp_encounter, ObjectAccessOperation.READ)`

Reply:
Thanks, point taken that's clearer. this has now been refactored a bit more with the has_permission_to_read being moved into the role

Reply:
There is an argument for keeping the `val is <True|False>` model in the sense that certain values in Python resolve to True in boolean expressions.  For example:
```
if None:
    print('hello')
```
Prints nothing, it is not a syntax error.

Likewise, 
```
if 'a':
    print('hello')
```
Prints 'hello'.  If we want to explicitly have our tests check the values returned are actually booleans, the `is` check would be appropriate.  If we simply want to ensure that the response value is *generally* True or False, we can simplify the assert.